FirebaseInstallations/Source/Library/Public/FIRInstallations.h

```diff
@@ -59,15 +59,17 @@ NS_SWIFT_NAME(Installations)
 
 /**
  * Returns a default instance of `Installations`.
- * @return Returns an instance of `Installations` for `FirebaseApp.defaultApp(). Throws an exception
- * if the default app is not configured yet.
+ * @return Returns an instance of `Installations` for `FirebaseApp.defaultApp().
+ * @throw Throws an exception if the default app is not configured yet or required  `FirebaseApp`
```

Comment:
Good recommendation, but I'll probably keep it as is to be consistent with the rest of the AppleDocs in the repo. Objective C exceptions are not handled usually, so the exception type is not so important and is mostly informative for a crash report reader. Because of that we use just an SDK domain for most of exceptions.

@ryanwilson do you have an opinion on it?

Reply:
Keeping it like this is fine I think - when rendered it's just a text field so having it start with "Throws" is fine.

Reply:
Sounds good, thanks for the explanation.
I quickly googles Objective C documentation and apparently there is no @throw keyword, but rather a @warning keyword. How about using that instead?

Reply:
`@throw` renders well for me:

<img width="524" alt="Screen Shot 2020-01-16 at 10 36 33 AM" src="https://user-images.githubusercontent.com/1841926/72538635-1b9a7a00-384c-11ea-8252-d206cf55e2de.png">